tests/test_api_periodictask.py

```diff
@@ -166,6 +167,7 @@ def test_01_crud(self):
         self.assertEqual(result_dict['nodes'], ['pinode1', 'pinode2'])
         self.assertEqual(result_dict['last_runs'], {})
         last_update = parse_timestamp(result_dict['last_update'])
+        self.assertEquel(result_dict['retry_if_failed'], False)
```

Comment:
Please fix the typo and ensure that the tests run successfully.

   assertEquel -> assertEqual

Reply:
Done in 969450a5